examples/40_paper/2018_kdd_rijn_example.py

```diff
@@ -8,9 +8,116 @@
 ~~~~~~~~~~~
 
 | Hyperparameter importance across datasets
-| Jan van Rijn and Frank Hutter
+| Jan N. van Rijn and Frank Hutter
 | In *Proceedings of the 24th ACM SIGKDD International Conference on Knowledge Discovery & Data Mining*, 2018
 | Available at https://dl.acm.org/citation.cfm?id=3220058
-
-This is currently a placeholder.
 """
+import fanova
+import json
+import logging
+import matplotlib.pyplot as plt
+import openml
+import pandas as pd
+import seaborn as sns
+
+root = logging.getLogger()
+root.setLevel(logging.INFO)
+
+##############################################################################
+# With the advent of automated machine learning, automated hyperparameter
+# optimization methods are by now routinely used in data mining. However, this
+# progress is not yet matched by equal progress on automatic analyses that
+# yield information beyond performance-optimizing hyperparameter settings.
+# In this example, we aim to answer the following two questions: Given an
+# algorithm, what are generally its most important hyperparameters?
+#
+# This work is carried out on the OpenML-100 benchmark suite, which can be
+# obtained by `openml.study.get_suite('OpenML100')`. In this example, we
```

Comment:
Unfortunately, this isn't markdown but rst and you need to use two backticks to get code rendering.